multiqc/modules/truvari/bench.py

```diff
@@ -0,0 +1,234 @@
+            "description": "Number of the comp VCF calls non-matching the base VCF",
+            "scale": "OrRd",
+            "format": "{:,.d}",
+            "min": 0,
+        }
+        keys["FN"] = {
+            "title": "FN",
+            "description": "Number of the base VCF calls non-matching the comp VCF",
+            "scale": "BuGn",
+            "format": "{:,.d}",
+            "min": 0,
+        }
+
+        # Reuse info from bench_headers but hide by default
+        keys["precision"] = dict(bench_headers["precision"], hidden=True)
+        keys["recall"] = dict(bench_headers["recall"], hidden=True)
+        keys["f1"] = dict(bench_headers["f1"], hidden=True)
+
+        keys["base cnt"] = {
+            "title": "Base",
+            "description": "Number of calls in the base VCF",
+            "scale": "Oranges",
+            "format": "{:,.d}",
+            "min": 0,
+        }
+        keys["call cnt"] = {
+            "title": "Call",
+            "description": "Number of calls in the comp VCF",
+            "scale": "Blues",
+            "format": "{:,.d}",
+            "min": 0,
+        }
+        keys["TP-call_TP-gt"] = {
+            "title": "TP-call with GT match",
+            "description": "TP-call with genotype match",
+            "hidden": True,
+            "scale": "Greens",
+            "format": "{:,.d}",
+            "min": 0,
+        }
+        keys["TP-call_FP-gt"] = {
+            "title": "TP-call w/o GT match",
+            "description": "TP-call without genotype match",
+            "hidden": True,
+            "scale": "YlGn",
+            "format": "{:,.d}",
+            "min": 0,
+        }
+        keys["TP-base_TP-gt"] = {
+            "title": "TP-base with GT match",
+            "description": "TP-base with genotype match",
+            "hidden": True,
+            "scale": "RdPu",
+            "format": "{:,.d}",
+            "min": 0,
+        }
+        keys["TP-base_FP-gt"] = {
+            "title": "TP-base w/o GT match",
+            "description": "TP-base without genotype match",
+            "hidden": True,
+            "scale": "Purples",
+            "format": "{:,.d}",
+            "min": 0,
+        }
+        keys["gt_concordance"] = {
+            "title": "GT concordance",
+            "description": "Genotype concordance. Definition: TP-call with GT / (TP-call with GT + TP-call w/o GT)",
+            "hidden": True,
+            "scale": "GnBu",
+            "suffix": "%",
+            "modify": lambda x: x * 100,
+        }
+
+        self.add_section(
+            name="Truvari bench",
+            anchor="truvari-bench",
+            description="Concordance statistics parsed from the output from <code>truvari bench</code>.",
+            plot=table.plot(data, keys, {"id": "truvari-bench-summary"}),
+        )
+        # Generate shuffled list of colors to label samples
+        colors = list(mcolors.CSS4_COLORS)
+        random.shuffle(colors)
```

Comment:
To make it reproducible, I'd pass a seed to random.

But even better, I'd use the `mqc_colour` module, pick a scale, and pass the sample index into it. See how it's done for Freyja: https://github.com/ewels/MultiQC/blob/256fd77e4e99ad0ba11cc6a609a9e7b839b1a3f0/multiqc/modules/freyja/freyja.py#L58
https://github.com/ewels/MultiQC/blob/256fd77e4e99ad0ba11cc6a609a9e7b839b1a3f0/multiqc/modules/freyja/freyja.py#L100

Reply:
I think I got too hanged up on not repeating colors, but with the tooltip this is not really a problem.  Now it uses the `mqc_colour` module instead of my bespoke solution ;)